src/lib/queries.js

```diff
@@ -0,0 +1,46 @@
+import CozyClient, { Q } from 'cozy-client'
+import { APPS_DOCTYPE, KONNECTORS_DOCTYPE } from 'doctypes'
+
+const THIRTY_SECONDS = 30 * 1000
+
+export const queryResultAppsAppPermissions = slugName => ({
+  definition: Q(APPS_DOCTYPE).getById('io.cozy.apps/' + slugName),
+  options: {
+    as: 'io.cozy.apps/' + slugName,
+    fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS),
+    singleDocData: true
+  }
+})
+
+export const queryResultKonnectorsAppPermissions = slugName => ({
+  definition: Q(KONNECTORS_DOCTYPE).getById('io.cozy.konnectors/' + slugName),
+  options: {
+    as: 'io.cozy.konnectors/' + slugName,
+    fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS),
+    singleDocData: true
+  }
+})
+
+export const queryResultRemoteRequestsAppPermissions = () => ({
+  definition: Q('io.cozy.remote.requests'),
```

Comment:
`'io.cozy.remote.requests'` should be added as a const here: https://github.com/cozy/cozy-settings/blob/master/src/doctypes/index.js
```suggestion
  definition: Q(REMOTE_REQUESTS_DOCTYPE),
```
And then,